django/db/migrations/utils.py

```diff
@@ -0,0 +1,28 @@
+from django.apps import apps
+from django.db.migrations.autodetector import MigrationAutodetector
+from django.db.migrations.loader import MigrationLoader
+from django.db.migrations.state import ProjectState
+
+
+def check_unmigrated_models(loader=None, connection=None):
+    """
+    :return: A (possibly empty) list of messages,
+    if the currently registered models have changes that are not
+    yet represented in a migration
+    """
+    messages = []
+    if loader is None:
+        loader = MigrationLoader(connection)
+
+    autodetector = MigrationAutodetector(
+        loader.project_state(),
+        ProjectState.from_apps(apps),
+    )
+    if autodetector.changes(graph=loader.graph):
+        messages.extend([
```

Comment:
I didn't actually expect, or mean for you to benchmark it, as either is "fast enough" IMHO. I simply didn't notice the trailing comma on L24, hence my assumption it was just one message (in which case *I'd* consider it more idiomatic to use `append`)